app/controllers/JavaListController.scala

```diff
@@ -80,24 +80,26 @@ class JavaListController @Inject() (
     ).descendingOrder.map(_.show)
 
   private val vendors = Map(
-    "adpt"    -> "AdoptOpenJDK",
-    "albba"   -> "Dragonwell",
-    "amzn"    -> "Corretto",
-    "gln"     -> "Gluon",
-    "grl"     -> "GraalVM",
-    "librca"  -> "Liberica",
-    "nik"     -> "Liberica NIK",
-    "none"    -> "Unclassified",
-    "open"    -> "Java.net",
-    "oracle"  -> "Oracle",
-    "mandrel" -> "Mandrel",
-    "ms"      -> "Microsoft",
-    "sapmchn" -> "SapMachine",
-    "sem"     -> "Semeru",
-    "tem"     -> "Temurin",
-    "trava"   -> "Trava",
-    "zulu"    -> "Zulu",
-    "zulufx"  -> "ZuluFX"
+    "adpt"       -> "AdoptOpenJDK",
+    "albba"      -> "Dragonwell",
+    "amzn"       -> "Corretto",
+    "gln"        -> "Gluon",
+    "graalvm-ce" -> "GraalVM Community",
+    "graalvm"    -> "Oracle GraalVM",
+    "grl"        -> "GraalVM",
```

Comment:
Is it technically possible and should "GraalVM" be renamed to something like "GraalVM (old)" to make it clear to users that `grl` is no longer updated? Maybe it's even possible to change the order and move the item to the end of the list?

Reply:
I think that should be done manually. /cc @marc0der

Reply:
We'll make the `grl` binaries hidden as they will soon be superceded by the new variants. We'll also remove the old category and replace it with the new ones.

Reply:
Didn't expect this to be possible but great!

Reply:
Okay, I've had a closer look and have some suggestions:
In the first column of the Java view, we have 14 characters to display the vendor name. Unfortunately, `GraalVM Community` is way longer and would get truncated. How about `GraalVM CE` instead? Then, to ensure that the listings appear beside each other, I'd suggest `GraalVM Oracle` instead of `Oracle GraalVM`.

In terms of the handles, I think we should keep with brevity (we currently identify GraalVM by `grl`). I would go for `graal-ce` for Community and perhaps `graal` for the Oracle variant. This is keeping in mind that this vendor string will be appended to the end of each version listed.

Reply:
Hi @marc0der,
Thanks for your suggestions! Some comments:
- "GraalVM CE" should be fine and is consistent with `graalvm-ce`.
- "GraalVM Oracle" should be fine as well, given this is the vendor table and it will improve the order
- I understand why you're aiming for short identifiers, `grl` just seems a bit too short for users to understand what it is. Also, I suggested a new name so that we avoid any versioning issues (the old 22.3 is greater than the new 20.0.1). While `graal` and `graal-ce` are a compromise, I suggested `graalvm` and `graalvm-ce` to avoid any confusion with the Graal Cloud Native (GCN) SDK (supported by SDKMAN) and any future projects released under the Graal umbrella. Although slightly longer, I'd still vote for `graalvm` and `graalvm-ce`.

Reply:
I've updated both names

Reply:
Hi guys, sorry, but just looking at this discussion again, I want to bring it back to something simpler.

These identifiers aren't meant to be descriptive to the users. They are merely identification slugs that differentiate various Java variants. The actual descriptions are provided by the Vendor column in view produced by `sdk list java`. Secondly, not one of these slugs has a dash, so I'm discarding that from the community edition's slug too.

I will opt for `graal` and `graalce` for consistency with what we currently have.

I'll also keep `grl` around for a little while until we have transitioned smoothly.

Reply:
Hi @marc0der, that's ok, just wanted to provide some input from our side. `graal` and `graalce` are fine and already an improvement over `grl`.

Reply:
Thanks @fniephaus,
I'll perform a release of these changes later today. Then at some point, we can ask @eddumelendez to run the disco integration and we should start seeing the newly published versions appear :tada: